Client.xcodeproj/xcshareddata/xcschemes/Ecosia.xcscheme

```diff
@@ -127,9 +127,6 @@
                <Test
                   Identifier = "CustomSearchEnginesTest">
                </Test>
-               <Test
-                  Identifier = "DefaultSearchPrefsTests">
```

Comment:
Do `DefaultSearchPrefsTests ` need to be deactivated? It seems you fixed them, didn't you?

Reply:
I did fix them, this is actually to re-activate 😅  I'm removing from `SkippedTests`